### DISCUSSION TITLE
Logisim-evolution stable release checklist

### DISCUSSION THREAD

**Discussion by MarcinOrlowski** [logisim-evolution/logisim-evolution#1004] [Dev talk]
Release flow (assuming all the development goes in `develop/` branch as it is now):

- [ ] branch `develop` as `rc`  - this allows work to continue in `develop/`
- [ ] when it is ready, merge `rc` branch into `master`
- [ ] Ensure `gradle.properties` version is correct for the release
- [ ] Ensure `CHAGELOG` lists version number and date of release (`YYYY-MM-DD` format)
- [ ] Commit changes it needed.
- [ ] build release version from `master` (so generated `BuildInfo` class will say `master/....`)
- [ ] `git tag -a VERSION` (i.e `v3.7.0`)
- [ ] `git push --follow-tags`
- [ ] Make release on Github page (+ paste `CHAGELOG` for that version into release information form)
- [ ] branch `master` into `<VERSION>` (i.e. `git checkout -b v3.7.0`) so we can easily cherrypick any features or bugfixes from develop

Anything missed?

**Comment by BFH-ktt1** [comment]
No I think that you covered all, the question that is open to me is do we want to build the "exe's" automatically or, as I did up to now, by hand on the different platforms.

**Reply by MarcinOrlowski** [reply]
That's good question. I think that we should have a github action worker that would be triggered if you merge anything into master and that worker would simply build and package HEAD of master.

**Reply by MarcinOrlowski** [reply]
Which should be basically copy-paste of nightly. Will take a look
